test/callbacks.js

```diff
@@ -0,0 +1,377 @@
+// The API key and IDs used in this file belong to the official Ghost Inspector API testing account.
+const assert = require('assert').strict
+const fs = require('fs')
+const GhostInspector = require('../index')('ff586dcaaa9b781163dbae48a230ea1947f894ff')
```

Comment:
Same here, not a huge fan of having a publicly-accessible API key, even in `test`.